lms/djangoapps/teams/views.py

```diff
@@ -81,7 +74,38 @@ def team_post_save_callback(sender, instance, **kwargs):  # pylint: disable=unus
                 )
 
 
-class TeamsDashboardView(View):
+class TeamAPIPagination(pagination.PageNumberPagination):
```

Comment:
I'm not sure why the Team API should use a different pagination schema than our other REST APIs. It seems to me that they all should return a count, the current page etc, so that clients can show details about the page being shown. Note that we built a shared Backbone pagination component that knows how to interpret these results, so it would be really beneficial if all of our APIs returned the same details. It would make documenting our APIs easier too.

Can this be made into a shared base class that we use for all our REST APIs? Is there something team-specific about it? Our intention was just to follow the REST API conventions and not to implement something unique here.